tests/ui/conftest.py

```diff
@@ -69,18 +61,34 @@ def selenium(selenium, request):
     Mobile size: 738x414 (iPhone 7+)
 
     """
-    # Skip mobile test with marker 'desktoponly'
-    if request.node.get_marker('desktoponly') and request.param == MOBILE:
+    # Skip mobile test with marker 'desktop_only'
+    if request.node.get_marker('desktop_only') and request.param == MOBILE:
         pytest.skip('Skipping mobile test')
     selenium.set_window_size(*request.param)
     return selenium
 
 
 @pytest.fixture
-def devhub_login(selenium, fxa_account):
+def fxa_account(request):
+    """Fxa account to use during tests that need to login.
+
+    Returns the email and password of the fxa account set in Makefile-docker.
+
+    """
+    try:
+        fxa_account.email = os.environ['FXA_EMAIL']
+        fxa_account.password = os.environ['FXA_PASSWORD']
+    except KeyError:
+        if request.node.get_marker('fxa_login'):
+            pytest.skip('Skipping test because no fxa account was found.')
```

Comment:
It may make sense to add something like "Are FXA_EMAIL and FXA_PASSWORD environment variables set?" to the message since in our case it is most likely an error and unexpected.